library/ssl_misc.h

```diff
@@ -2172,4 +2212,15 @@ static inline int psa_ssl_status_to_mbedtls( psa_status_t status )
 }
 #endif /* MBEDTLS_USE_PSA_CRYPTO || MBEDTLS_SSL_PROTO_TLS1_3 */
 
+#if defined(MBEDTLS_ECDH_C)
+
+int mbedtls_ssl_tls13_read_public_ecdhe_share( mbedtls_ssl_context *ssl,
+                                               const unsigned char *buf,
+                                               size_t buf_len );
+
+#endif /* MBEDTLS_ECDH_C */
+
+int mbedtls_ssl_tls13_cipher_suite_is_offered( mbedtls_ssl_context *ssl,
```

Comment:
Please implement it as static inline function.

Reply:
Is there one standard that what function should be implemented with inline?

Reply:
Anyway, I am not sure whether there is standard, but I think we ccan implememt it as static function in ssl_misc.h temp.